src/main/java/duke/Task.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import javax.print.DocFlavor;
+import java.io.Serializable;
+
+public class Task implements Serializable {
+    private String name;
+    private boolean status;
```

Comment:
Shouldn't this variable be named to sound like a boolean?

Maybe you could rename it to something like `isDone`